CoC.md

```diff
@@ -3,10 +3,10 @@
 ## Preamble
 
 Like the broader technical and activism communities as a whole, the SecureDrop
-team is made up of a mixture of staff and volunteers from all over the world,
-working on every aspect of our mission—including operations, software
+Community is made up of a mixture of staff and volunteers from all over the world,
+engaged in various activities—including operations, software
 development, mentorship, and building connections with great people and
-organizations.
+organizations. Every individuals who claim to be part of the SecureDrop Community are on an equal footing, regardless of their affiliation to an organization. There is no hierarchy in the SecureDrop Community, we are organized horizontally.
```

Comment:
> "All individuals who associate themselves with the SecureDrop Community are on equal footing, regardless of their affiliation with any specific organization."

:+1:

re: whether or not we are hierarchical, for the sake coming to consensus on this CoC content, it might be wise to avoid statements about governance/how we are organized (because it's going to likely involve longer debate) and keep it scoped to just CoC/anti-harassment policy

For inspiration, in the long term of SecureDrop project, eventually I imagine we will have other documents, e.g. like Tor does, with the CoC scoped to what constitutes appropriate/inappropriate behavior/how you escalate problems, and then other documents describing other community policies (around governance, voting, who is a member of the community): https://gitweb.torproject.org/community/policies.git/tree/

Reply:
So even though I like the idea of flattening things out and decentralizing, I think that goes outside the scope of the CoC. I think once this is one, this should be the first document in a `community` git repo for SD. Maybe hosted on the `.club` site.